sources/packages/web/src/components/common/students/StudentApplications.vue

```diff
@@ -157,6 +155,21 @@ export default defineComponent({
       type: Number,
       required: false,
     },
+    enableViewApplicationBtn: {
```

Comment:
For properties in general I would avoid adding the component type and use an abbreviation like btn or similar.
IMO these three properties could be named as.
- `enableViewApplication`
- `allowManageApplication`
- `displayApplicationName`

Reply:
Added `btn` only because of the last property, the last property is the `view application` action in the name. 
![image](https://user-images.githubusercontent.com/77353155/235013640-8a783429-72c9-43ab-9174-1456bdd68523.png).

`displayApplicationName` is confusing. do you guys have a better name, I can update it